en/micro-integrator/docs/references/connectors/kafka-connector/kafka-connector-producer-example.md

```diff
@@ -0,0 +1,120 @@
+# Kafka Connector Example
+
+Given below is a sample scenario that demonstrates how to send messages to a Kafka broker via Kafka topics.The publishMessages operation allows you to publish messages to the Kafka brokers via Kafka topics.
+
+## What you'll build
+Given below is a sample proxy service that illustrates how you can connect to a Kakfa broker with the `init` operation and then use the `publishMessages` operation to publish messages via the topic.
+
+**publishMessages**
```

Comment:
I wonder if this synapse logic part should come under `What you'll build` section. @samgnaniah - I think we should remove this section from here as per our doc structure?